README.md

```diff
@@ -45,9 +45,6 @@ The instrumentations in this repo are:
 
 
 ## Compatibility with opentelemetry versions
-### Instrumentations in this repo are using opentelemetry [Instrumentation API](https://github.com/open-telemetry/opentelemetry-js/tree/main/packages/opentelemetry-instrumentation).
```

Comment:
I would just call this section "Compatability Table" directly

Reply:
done